Drop-In/src/test/java/com/braintreepayments/api/DropInActivityUnitTest.java

```diff
@@ -778,4 +795,15 @@ private void assertExceptionIsReturned(String analyticsEvent, Exception exceptio
         assertEquals(exception.getClass(), actualException.getClass());
         assertEquals(exception.getMessage(), actualException.getMessage());
     }
+
+    private BrowserSwitchResult createBrowserSwitchSuccessResult() {
+        int requestCode = 123;
+        Uri url = Uri.parse("https://example.com");
+        JSONObject metadata = new JSONObject();
+        String returnUrlScheme = "sample-scheme";
+
+        BrowserSwitchRequest browserSwitchRequest =
+            new BrowserSwitchRequest(requestCode, url, metadata, returnUrlScheme, true);
```

Comment:
```suggestion
            new BrowserSwitchRequest(requestCode, url, metadata, returnUrlScheme);
```
I don't remember what that true parameter was for, but looks like that method was changed in browser switch